Gemfile.lock

```diff
@@ -51,100 +35,106 @@ GEM
     guard-rake (1.0.0)
       guard
       rake
-    hiera (1.3.4)
+    hiera (3.2.0)
       json_pure
-    highline (1.6.21)
-    hitimes (1.2.2)
-    json (1.8.2)
-    json_pure (1.8.2)
+    highline (1.7.8)
+    json (2.0.2)
+    json_pure (2.0.2)
     launchy (2.4.3)
       addressable (~> 2.3)
-    listen (2.8.5)
-      celluloid (>= 0.15.2)
-      rb-fsevent (>= 0.9.3)
-      rb-inotify (>= 0.9)
-    lumberjack (1.0.9)
+    listen (3.1.5)
+      rb-fsevent (~> 0.9, >= 0.9.4)
+      rb-inotify (~> 0.9, >= 0.9.7)
+      ruby_dep (~> 1.2)
+    lumberjack (1.0.10)
     metaclass (0.0.4)
-    metadata-json-lint (0.0.6)
+    metadata-json-lint (0.0.11)
       json
       spdx-licenses (~> 1.0)
     method_source (0.8.2)
     mocha (1.1.0)
       metaclass (~> 0.0.1)
-    multi_json (1.10.1)
+    multi_json (1.12.1)
     multipart-post (2.0.0)
-    nenv (0.2.0)
+    nenv (0.3.0)
     net-http-persistent (2.9.4)
     net-http-pipeline (1.0.1)
-    notiffany (0.0.4)
+    notiffany (0.1.1)
       nenv (~> 0.1)
       shellany (~> 0.0)
-    pry (0.9.12.6)
-      coderay (~> 1.0)
-      method_source (~> 0.8)
+    pry (0.10.4)
+      coderay (~> 1.1.0)
+      method_source (~> 0.8.1)
       slop (~> 3.4)
-    puppet (3.7.4)
-      facter (> 1.6, < 3)
-      hiera (~> 1.0)
+    puppet (4.5.3)
```

Comment:
in travisci , we actually delete the .lock file and run bundle install again depending on the version of puppet currently set. So this is again tested. But I will add the puppet version 3.7 in the travis build matrix